pyproject.toml

```diff
@@ -7,10 +7,7 @@ requires = [
 build-backend = "jupyter_packaging.build_api"
 
 [tool.hatch.build]
```

Comment:
Oh, this hatch config can be removed.  `jupyter_packaging` uses setuptools, so the correct fix was the one you made in `setup.py`.

Reply:
just removed `[tool.hatch.build]` and it continue to build fine on my env.